Automattic/wp-calypso
# Copy Site: Add back functionality on domains step

#### Proposed Changes

Add back functionality on domains step

#### Testing Instructions



1. Navigate to `/sites`
2. Copy a site.
3. You will be navigated to the domains selection screen
4. Make sure that a back button exists in the top right corner.
5. Click the back button
6. Make sure you'll be navigated back to `/sites`

![back_button_domains_step](https://user-images.githubusercontent.com/497103/213423935-a3033579-be7d-4c51-9dff-411f282d2988.png)

#### Pre-merge Checklist





Fixes https://github.com/Automattic/dotcom-forge/issues/1519
